docs/migration/fastapi.rst

```diff
@@ -232,7 +234,7 @@ preferred way of handling this is extending :doc:`/usage/security/abstract-authe
 ..  seealso::
 
     To learn more about security and authentication, check out this chapter in the
-    documentation: `Security <usage/8-security/0-intro/>`_
+    documentation: :doc:`/usage/security/index`
```

Comment:
If we are doing the bulleted list in this PR, it would make sense to do it here too?

Reply:
Yes that makes sense for consistency, I'll go through and update other `seealso` sections to lists.